freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -65,12 +70,90 @@ def handle(self, event):
         self.log_info(
             "Orignial nvrs of build in the advisory #{0} are: {1}".format(
                 db_event.search_key, " ".join(original_nvrs)))
+        # Get images by nvrs and then get their digests
+        original_images_digests = set(self.get_digests_by_nvrs(original_nvrs))
+        if not original_images_digests:
+            log.warning("The are no digests for NVRs: %s",
+                        ",".join(original_nvrs))
+            return []
+
+        index_images = self._pyxis.get_operator_indices()
+        # get latest bundle images per channel per index image filtered by higher version
+        all_bundles = self._pyxis.get_latest_bundles(index_images)
 
-        msg = "Skipping due to being blocked on further implementation for now."
+        bundles = self._pyxis.filter_bundles_by_digests(original_images_digests,
+                                                        all_bundles)
+        bundle_digests = []
```

Comment:
Could you please make this a set? It's common that a bundle image version is available in multiple index images (e.g. a bundle that can be installed on OCP 4.6 and OCP 4.7).